certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+            "type": "string",
+            "default": "69a489d9-53fd-43aa-b04c-5c15ed4c73dd",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "UserToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "98ae7807-0902-4e66-9b0a-bfe0ecd204de",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "id",
+                  "title": "",
+                  "x-ms-visibility": "internal"
+                },
+                "typeName": {
+                  "type": "string",
+                  "description": "typeName",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "Product.Purchased"
+                },
+                "filters": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "title": "Product Code",
+                    "x-ms-visibility": "advanced"
+                  },
+                  "description": "filters"
+                },
+                "secret": {
+                  "type": "string",
+                  "description": "secret",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "prodPurchasedFLOW"
+                },
+                "callBackUrl": {
+                  "type": "string",
+                  "description": "callBackUrl",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "x-ms-notification-url": true
+                },
+                "description": {
+                  "type": "string",
+                  "description": "description",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "AddedByFlow"
+                }
+              },
+              "required": [
+                "callBackUrl",
+                "description",
+                "secret",
+                "typeName"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {},
+  "security": [],
+  "tags": [],
+  "x-ms-connector-metadata": [
+      {
+        "propertyName": "Website",
+        "propertyValue": "http://wwww.impexium.com"
+      },
+      {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://www.impexium.com/impexium-information-security-policy/"
+      },
+      {
+        "propertyName": "Categories",
+        "propertyValue": "AMS;Membership;Association Management Software"
```

Comment:
@RjImpx Please add maximum two categories only, selected from the pre-defined list mentioned in the [instructions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#product-or-end-service-metadata)